cfgov/v1/jinja2/v1/_includes/macros/filters.html

```diff
@@ -99,6 +99,7 @@
     'action':                   '/',
     'expand_label':             'Filter',
     'submit_label':             'Apply filters',
+    'form_method':              'get',
```

Comment:
Add entry for `user_options.form_method` to the `user_options` code comment docs [here](https://github.com/cfpb/cfgov-refresh/blob/security-fixes/cfgov/v1/jinja2/v1/_includes/macros/filters.html#L60)